### ISSUE TITLE
Excavator command shell keeps restarting

### ISSUE THREAD

**Issue by tomcleaver** [nicehash/NiceHashMiner-Archived#1407]
Hi guys, i'm having a problem that has only recently started happening, i've done a fresh install of windows and ensured the graphics drivers are all installed 100% correctly, i'm running the following equipment:

11 x GTX 1080ti
1 x GTX 1080
16GB RAM (2 x 8GB)
Core i5 7200 Processor (Quad Core)
ASUS B250M Mining Expert Motherboard (All drivers/chipset/bios up to date)
3 x 1250W PSU so there's plenty of extra power to go around.
Nvidia GeForce Graphics Driver 388.17
Virtual Memory is set to 240GB initial, 284GB Max

This is happening without any overclocking.

The issue i'm having is that for seemingly no reason the excavator command shell window abruptly restarts every 10 seconds or so, drastically reducing my mining efficiency. I am also unable to click the stop mining button until after the window re-opens

Please help, i've done everything I can think of to correct the issue to no avail (this includes uninstalling everything, including the visual C++ runtimes, graphics drivers using DDU, and even reinstalled windows entirely).

At the time of writing i see that nvidia has released a new driver, i'll try installing that, will update if it helps at all.
  
Also, sometimes the miner keeps on doubling up the projected earnings, i.e. it keeps re-adding the set of devices and their projected payout, it continues indefinitely until I close the nicehash miner by ending the task via task manager.

**Comment by tomcleaver** [comment]
No luck from the graphivcs driver update, also same problem still exists after the NHM 2.0.1.6.

Any advice devs?
Need anything to help diagnose the issue?

Some help would be appreciated.

**Comment by russalex83** [comment]
Having the same problem...
10x - EVGA GTX1080's, 1x - 1060, running on ASUS B250 board. NiceHash miner 2.0.1.6, latest NVidia driver.
Not sure what to do, keeps restarting every 10sec or so, all GPUs running below 60.
 Please help!

**Comment by russalex83** [comment]
Any solutions ? Anyone?

**Comment by tomcleaver** [comment]
Yeah man, I gave up on waiting for a response from NH, their software is shit, completely lags my system.

You can use awesomeminer and just select nicehash as the managed profit miner pool, enter your nicehash wallet address (or external wallet address) and go.

Have been using it for the last two days and there is no decrease in profitability.

And even better you can benchmark all your devices at the same time.

www.awesomeminer.com

**Comment by russalex83** [comment]
Hey man, did anyone tell you that you are a life saver 👍 🥇 :). After spending 2 nights until 5am fighting with stupid NH I almost burn out my system. With the latest Nvidia driver NH was crashing complete system. Made it work with 2 month old driver but stupid NH Excavator kept restarting still leaving me with no profit. Tried reinstalling windows and NH few times waiting for STUPID benchmarks (GPU optimization). Was thinking bad hardware, wanted to go through each card... etc ...
- awesomeminer  is the answer, already running with no issues.
Just keep thanking God for people like you!

Thanks a lot man!

**Comment by tomcleaver** [comment]
No problem, happy to help.

There's one thing you can do to slightly improve the setup though, if your equihash algo is set to use the EWBF that comes with awesomeminer, download the no dev fee version here: 
https://bitcointalk.org/index.php?topic=2007661.0

Unzip the nofee EWBF to its own install location and add folder via the "upload custom software" found in the tools tab. Then select the folder and .exe files in the next dialogue. Then go to Options under the main tab, select Profit Profiles on the left, select Nvidia GPU. Under the profiles window move EWBF CUDA Zcash miner to the top of the list. Now click on it and open the configure page. Now select specify existing installation path and point the Path to your .exe file. Voila, No more devfee when mining equihash!

Cheers!

**Comment by russalex83** [comment]
Sure, I'll try this!

Thanks again!

**Comment by russalex83** [comment]
BTW, by following NH's Profitability calculator : https://www.nicehash.com/profitability-calculator/nvidia-gtx-1080?e=0.13&currency=USD the estimated profit runs from $5-5.9/day from one GTX1080 and that's pretty much what I'm getting so far with awesomeminer and NH wallet, sometimes slightly more, slightly less. You mentioned about: "No more devfee when mining equihash!". Are there any unknown fees that they charge us? Should I be making more then Profitability calculator says? I know you have one GTX1080 as you mentioned and can compare your earnings on this card against NH estimate. How accurate are they? Sorry, I just got into mining and still a newbie :). Still learning the process and market. Please clarify when you'll get a chance.
Thanks in advance!

**Comment by tomcleaver** [comment]
The EWBF miner has a devfee built in, it takes 5% of your shares, ie. it switches mining to the devs wallet a certain amount of time.

Also, reduce the profit switiching interval to 1 minute  and 1% (this will make sure it's always switching to the most profitable. (Options > Profit Switching)

**Comment by russalex83** [comment]
I figured now. Thanks for your advice, I saw a couple bucks difference with no-devfee version when specifically mining with EWBF for few hours and disabling ccminer which is great! But I get slightly better profit with ccminer then EWBF with my current setup so I just let the application to do its job. Awesome Miner seems a lot more stable and reliable then NH, but I still get random system reboots (don't know the reason yet, but I'm ok with this for now. Luckily I made 11 Nvidia GPU's running simultaneously on Asus B250 Mining Expert which kind of a miracle for me since Asus claims that they only support up to 8 Nvidia Cards: https://www.asus.com/us/support/FAQ/1034921/ I will order another piece of RAM to make it 16Gb and increase Virtual memory as well. Will see if it helps. Not sure what could cause my system random crash but I guess I have to figure it out in a long run. Did you experience any similar issues with you system? Any suggestions?  Thanks!

**Comment by CRYTOFARMER876** [comment]
GUYS GOOD DAY the solution to this problem is right here below if it helps you feel free to support me via paypal or my cryto address  below. i will take credit for the below info as i was experiencing the same issues for some time. I even leave my computer mining for 2 weeks only to come back and find it stuck at the above error just after i leave my home- 2 weeks out of wasted time and money =big lost. lets get into the solution now this error is cause my microsoft security suite which protect your computer from hacker etc .. just like any protection you use eg-a condom when having sex so u know if you go bare back you will most likely able to catch some disease if the person is not clean,that being said lets fix this problem permanently forever now that you understand the risk. the fix is open your over clocking software either after burner or evga oc and set your core clock to +105 and your memory clock to +537 thats it your issues are done. if you get the excavator error or windows block graphics hardware , disable windows firewall for all profile and your good and add the excavotor.exe app to your firewall list. if this help denate to my payapl @janoy876@gmail.com and my btc wallet 3Bk54XPfRpYHgKQQuH1w9RAaApnzz5Y5Wq. please share my info as it will help others
